dashboard/app/views/devise/registrations/edit.html.haml

```diff
@@ -177,16 +177,20 @@
       %span#change-user-type-status
       %button.btn#change-user-type-button{disabled: true}= t('signup_form.user_type_button')
 
-%hr.danger.account-page-section-break
-%h2.danger= t('user.delete')
```

Comment:
Do we no longer show the horizontal rule and header for the Delete Account section in the case where the user cannot delete their own account?

Reply:
😱 thanks for catching that! i think i resolved a merge conflict incorrectly. i'll add that back